docs/sources/investigate/analyze-tracing-data.md

```diff
@@ -0,0 +1,89 @@
+---
+description: Analyze tracing data using comparison, root cause analysis, Comparison, and traces view to investigate trends and spikes.
+canonical: https://grafana.com/docs/grafana/latest/explore/simplified-exploration/traces/investigate/
+keywords:
+  - Traces Drilldown
+  - Analyze
+title: Analyze tracing data
+menuTitle: Analyze tracing data
+weight: 500
+---
+
+# Analyze tracing data
+
+To further analyze the filtered spans, use the dynamically changing tabs, **Comparison**, **Structure**, **Root cause analysis**, and **Trace list**.
+
+When you select a RED metric, the tabs underneath the metrics selection changes match the context.
```

Comment:
This sentence isn't reading properly to me. Is there an issue with agreement?

Reply:
```suggestion
When you select a RED metric, the tabs change to match the context.
```